packages/account/src/Sections/Verification/ProofOfOwnership/Card.jsx

```diff
@@ -6,8 +6,14 @@ import PropTypes from 'prop-types';
 import paymentMethodConfig from './payment-method-config';
 
 const Card = ({ card, handleChange, handleBlur, values, setFieldValue, index, error, validateField, updateErrors }) => {
+    const [show_button, setShowButton] = React.useState([]);
```

Comment:
can we have a better naming for this?
its not clear what it going to do!

Reply:
Hello @ali-hosseini-fs. Sure I'll change this to `show_browse_button`